### PR TITLE
SDL-0240 Revision - WebEngine support for SDL JavaScript 

### PR DESCRIPTION
**Pull request by kshala-ford**
This proposal is adding a new transport to the SDL JavaScript library to support (progressive) web apps to run on a WebEngine or a browser.

### CODE REVIEW

**Comment by kshala-ford** [comment]
It's a suggested revision to the proposal removing the chapter around the open HMI. This chapter is now introduced in #809. Applying this revision is accepted by the author only together with moving #809 into review.

**Comment by kshala-ford** [comment]
@theresalech the PR is ready for review.

File: proposals/0240-sdl-js-pwa.md [hunk 1]
```diff
@@ -109,65 +109,11 @@ The HMI API extension is mostly a copy of the cloud-app-properties included in t
...
-  o	number of lines of text
-  o	Complexity of use cases (number of steps to fulfill a use case)
-
-More items may be included in the ruleset as they become Driver Distraction affected.
-
-##### 2.2.4 System context and event change
+#### 1.5 App presentation with Templates
 
-Independent of the app presentation type, the HMI will continue to provide system context information from the app. An application which uses the open HMI should continue to recieve `OnHMIStatus` notifications and SDL Core will still be notified about event changes.
+The default app presentation approach should be template based. With the web app becoming active on the HMI, the HMI should present the application template. The body of the index.html file of the web app should be empty and should not be modified with scripts as the app won't be visible on the HMI anyways. The HMI should show the system component responsible for SDL templates. This component would receive the `Show` RPCs provided by the app.
```
Comment (theresalech) [inline proposals/0240-sdl-js-pwa.md]:
```suggestion
The default app presentation approach should be template based. With the web app becoming active on the HMI, the HMI should present the application template. The body of the index.html file of the web app should be empty and should not be modified with scripts as the app won't be visible on the HMI anyway. The HMI should show the system component responsible for SDL templates. This component would receive the `Show` RPCs provided by the app.
```

**Comment by kshala-ford** [comment]
@theresalech just applied the change. I was hoping to vote for merging the PR in the next Steering Committee meeting. As this PR is only removing items and not modifying remaining pieces I suppose the proposal can stay in review after merging (so return for revision with immediate return for review as already happened in other proposals before).

**Comment by theresalech** [comment]
@kshala-ford yes, we can ask the Steering Committee if they approve returning SDL 0240 for revisions, and bringing the revised proposal back into review tonight, as long as we are keeping the number of proposals in review to 6 or fewer.

**Pull request merged**